mlir/include/mlir/Dialect/Linalg/IR/LinalgInterfaces.td

```diff
@@ -54,6 +54,14 @@ def LinalgContractionOpInterface : OpInterface<"ContractionOpInterface"> {
       return $_op.getOperation()->getOperand(1);
     }]>,
     InterfaceMethod<
+    /*desc=*/"Returns the result value.",
+    /*retTy=*/"OpResult",
+    /*methodName=*/"res",
+    /*args=*/(ins),
+    /*methodBody=*/[{
+      return $_op.getOperation()->getResult(1);
```

Comment:
What is result 0?

Reply:
Ah, it should be 0. A unit test would catch this : ) Fixed.